processor/tailsamplingprocessor/processor.go

```diff
@@ -139,6 +139,9 @@ func getPolicyEvaluator(logger *zap.Logger, cfg *PolicyCfg) (sampling.PolicyEval
 	case Composite:
 		rlfCfg := cfg.CompositeCfg
 		return getNewCompositePolicy(logger, rlfCfg)
+	case And:
```

Comment:
It would be good to have a test case validating that this has been fixed.

Reply:
I modified the `TestCreateDefaultConfig` to use the testdata yml, in the test data we have all types of policies.